### ISSUE TITLE
Forge bucket models do not have a mask

### ISSUE THREAD

**Issue by pupnewfster** [MinecraftForge/MinecraftForge#6377]
**Minecraft Version:** 1.15.1

**Forge Version:** 30.0.11

**Steps to Reproduce:**
 1. Create a model for a bucket using the example on giga's model PR
 2. For example:
```json
{
  "parent": "forge:item/bucket",
  "loader": "forge:bucket",
  "fluid": "mekanism:brine"
}
```
 3. Get this message printed to the log on startup: `[minecraft/AtlasTexture]: Using missing texture, unable to load forge:textures/item/mask/bucket_fluid.png : java.io.FileNotFoundException: forge:textures/item/mask/bucket_fluid.png`

**Description of issue:**
I believe this has to do with the fact that the base bucket [model](https://github.com/MinecraftForge/MinecraftForge/blob/1.15.x/src/main/resources/assets/forge/models/item/bucket.json) and the bucket drip [model](https://github.com/MinecraftForge/MinecraftForge/blob/1.15.x/src/main/resources/assets/forge/models/item/bucket_drip.json) reference `forge:item/mask/bucket_fluid` and `forge:item/mask/bucket_fluid_drip` respectively. There is no `textures/item/mask/` in the 1.15 branch the closest I can find are [these](https://github.com/MinecraftForge/MinecraftForge/tree/1.15.x/src/main/resources/assets/forge/textures/items) in `textures/items` I am making an issue instead of a PR in part because I am unsure if @gigaherz just didn't rename things or forgot to upload the new masks in general, given I recall him having to modify the masks some when he was working on his model PR.

**Comment by gigaherz** [comment]
The patch file I used to copy over the changes into 1.15 must have lacked the mask images. I'll fix as soon as I get home.

**Comment by gigaherz** [comment]
Fixed in 503d2770
